deepchem/models/graph_models.py

```diff
@@ -495,6 +504,115 @@ def default_generator(self,
         ], [y_b], [w_b])
 
 
+class GraphConvKerasModel(tf.keras.Model):
```

Comment:
If this is meant to be an internal class, should there be a `_` at the start of the name?

Reply:
Good suggestion. Will put an `_` at the start.